gym_electric_motor/physical_systems/mechanical_loads/polynomial_static_load.py

```diff
@@ -47,18 +53,26 @@ def __init__(self, load_parameter=None, limits=None, load_initializer=None):
         self._c = self._load_parameter['c']
 
     def _static_load(self, omega, *_):
-        """
-        Calculation of the load torque for a given speed omega.
-        """
-        sign = 1 if omega > 0 else -1 if omega < 0 else 0
-        return sign * (self._c * omega**2 + self._b * abs(omega) + self._a)
+        """Calculation of the load torque for a given speed omega."""
+        sign = 1 if omega > 0 else -1 if omega < -0 else 0
+        # Limit the constant load term 'a' for velocities around zero for a more stable integration
+        a = sign * self._a \
+            if abs(omega) > self._a / self._j_total * self.tau_decay \
+            else self._j_total / self.tau_decay * omega
+        return sign * self._c * omega**2 + self._b * omega + a
 
     def mechanical_ode(self, t, mechanical_state, torque):
         # Docstring of superclass
-        return np.array([(torque - self._static_load(mechanical_state[self.OMEGA_IDX])) / self._j_total])
+        omega = mechanical_state[self.OMEGA_IDX]
+        static_load = self._static_load(omega)
+        load = torque - static_load
```

Comment:
I do not like that we call the sum of torques the "load" here.
I would rather suggest to distinguish "drive_torque", "load_torque" and "total_torque" (or something along those lines).

Reply:
A good point, thank you. 
Solely, I will keep the name of the input as torque because is inherited by the base-class "mechanical load". Furthermore in the documentation the input torque is documented correctly.